main_test.go

```diff
@@ -44,51 +89,230 @@ webhook:
 
 	certOverWriteConfig := filepath.Join(tmpDir, "cert-overwrite.yaml")
 	if err := os.WriteFile(certOverWriteConfig, []byte(`
-apiVersion: config.kueue.x-k8s.io/v1alpha1
+apiVersion: config.kueue.x-k8s.io/v1alpha2
 kind: Configuration
+namespace: kueue-system
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: :8080
 leaderElection:
   leaderElect: true
-enableInternalCertManagement: false
+  resourceName: c1f6bfd2.kueue.x-k8s.io
+webhook:
+  port: 9443
+internalCertManagement:
+  enable: true
+  serviceName: kueue-tenant-a-webhook-service
```

Comment:
It should be webhookServiceName then. Also secretName.

Reply:
Yes, that's right.

Reply:
Done.